### DISCUSSION TITLE
Welcome to Khoj!

### DISCUSSION THREAD

**Discussion by debanjum** [khoj-ai/khoj#39] [Announcements]
## 👋 Welcome!
  We’re setting up a forum for users of Khoj to connect. We hope that you:
  * 🤔 Ask any questions around setup, usage etc
  * 💡 Share your ideas and use-cases for Khoj
  * 🤝 Help other folks onboard to and explore Khoj 
  * 😇 Be nice and open-minded. Remember that a welcoming community is built together 💪.

Currently, it's just a few of us who are using (and developing) Khoj.
But this forum should help more folks discover, explore and improve Khoj together! 👯

**Comment by Aditya6721** [comment]
Dude I am new on github i just randomly found this post and i am trying to figure out what to do in it, I only have a phone is there any use of github on phone?
